### PR TITLE
Ensured documents for meetings are sorted by number

### PR DESCRIPTION
**Pull request by dvanleijenhorst**
Solves issue #834. 

Unnumbered items are currently sorted alphabetically underneath the numbered items, and numbered items are sorted as expected.

### CODE REVIEW

**Comment by dvanleijenhorst** [comment]
As a side-effect of trying to fix codeclimate issues, unnumbered items now appear sorted above the numbered items, and numbered items are as normal

**Comment by Koen1999** [comment]
This resolves #834

File: module/Decision/src/Decision/Model/Meeting.php [hunk 1]
```diff
@@ -214,4 +214,29 @@ public function addDocuments($documents)
...
+            $aa = preg_split("/(\.|\s)/", $a->getName());
+            $bb = preg_split("/(\.|\s)/", $b->getName());
+            for ($i = 0; $i < min(count($aa), count($bb)); $i++) {
+                if (!is_numeric($aa[$i])) {
+                    return -1;
+                } elseif (!is_numeric($bb[$i])) {
+                    return 1;
+                } elseif ($aa[$i] != $bb[$i]) {
+                    return $aa[$i] < $bb[$i] ? -1 : 1;
+                }
```
Comment (kokx) [inline module/Decision/src/Decision/Model/Meeting.php]: Using `else` generally decreases readability in methods. Instead, I would rewrite it into:

```php
if (!is_numeric($aa[$i])) {
    return -1;
}
if (!is_numeric($bb[$i])) {
    return 1;
}
if ($aa[i] != $bb[i]) {
    return $aa[$i] < $bb[i]$ ? -1 : 1;
}
```

Also. This function seems like it's basically lexicographic sort on document names. Would sort on `$doc->getName()` not be good enough?

Comment (jszanto) [reply module/Decision/src/Decision/Model/Meeting.php]: That would break when there are more than 9 documents
1
10
11
12
2
3
4
5
6

**Pull request merged**